src/main/webapp/script.js

```diff
@@ -272,9 +258,8 @@ function clearSavedProfileStatus() {
   profileStatusElement.style.display = "none";
 }
 
-
+/** Function gets recipe information from user input ID and displays the title on the page */
 function getRecipe(){
-  /** Function gets recipe information from user input ID and displays the title on the page */
   var numRecipe = document.getElementById("num-recipe").value;
```

Comment:
to verify: numRecipe refers to recipe id? Consider renaming to distinguish between the concept of number of recipe to return/query.